src/main/java/org/jvnet/hudson/test/TestExtensionLoader.java

```diff
@@ -64,7 +70,13 @@ protected boolean isActive(AnnotatedElement e) {
 
         TestExtension a = e.getAnnotation(TestExtension.class);
         if (a==null)        return false;   // stale index
-        List<String> testNameList = Arrays.asList(a.value());
+        List<String> testNameList;
+        try {
+            testNameList = Arrays.asList(a.value());
```

Comment:
Actually it is possible to use reflective calls to access `value` as either a `String` or a `String[]`. But there could only be a mismatch when you are loading a `*-tests.jar` from another plugin which was built with an older parent. And in this case any `@TestExtension`s in that other plugin would not be loaded in this one anyway.